src/BotPlutusInterface/Config/Types.hs

```diff
@@ -0,0 +1,62 @@
+module BotPlutusInterface.Config.Types (
+  ToValue (..),
+  sectionWithDefault,
+  sectionWithDefault',
+  withNamePrefixSpec,
+  serialize,
+  deserialize',
+  deserialize,
+) where
+
+import Config
+import Config.Schema
+import Config.Schema.Load.Error
+import Control.Exception (displayException)
+import Data.Bifunctor (first)
+import Data.Maybe (fromMaybe)
+import Data.Text (Text)
+import Data.Text qualified as Text
+import Text.PrettyPrint (Style (..), render, renderStyle, style)
+import Prelude
+
+class ToValue a where
+  toValue :: a -> Value ()
+
+sectionWithDefault ::
+  ToValue a =>
+  HasSpec a =>
```

Comment:
can we tuple these together?

Reply:
Do you mean type alias like: `type Configurable a = (ToValue a, HasSpec a)` or something like: `class Configurable a where confSpec ... confToValue ...` to avoid two classes implementation?